recipes/libpq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: libpq
+    version: {{ version }}
+
+build:
+  number: 0
+  skip: True   # [win]
+  # features:
```

Comment:
If you are going to try Windows right after feedstocking we can leave these here. No need to remove them now.